### PR TITLE
Removed need for mcrypt

### PR DESCRIPTION
**Pull request by jplourde5**
Forked the Anesidora project and replaced mcrypt with standard node.js crypto.  You may want to setup your own fork like I think you did with fuse.

### CODE REVIEW

**Comment by jplourde5** [comment]
Hold on this for a moment.  I'm doing some more testing and it burped.

**Comment by jplourde5** [comment]
I think that I just hit it too soon after restarting.  Did further testing on both Mac and RPI3 with no issues.

**Comment by jplourde5** [comment]
I also removed unneeded 'success' from the resolve statements in pandora.js so that the standard status results get returned

**Comment by dlom** [comment]
Updated anesidora to 1.2.0 following @jplourde5's PR, which removes the mcrypt dep

File: lib/actions/pandora.js [hunk 1]
```diff
@@ -37,15 +37,15 @@ function pandora(player, values) {
...
                  } else {
                    reject('No match found for ' + name);
                  } 
                }
              });
            }
         });
-    }, Promise.resolve('success')); 
+    }, Promise.resolve()); 
   }
```
Comment (jishi) [inline lib/actions/pandora.js]: What does this second argument do? I don't think Promise take two arguments?

Comment (jishi) [inline lib/actions/pandora.js]: You should also probably extract the login to a separate function that would return a promise, then you could make a much cleaner promise chain in playPandora. 

I can fix it though if I'm being unclear after merge.

**Comment by jplourde5** [comment]
mcrypt has now been removed and replaced with crypto in the main Anesidora branch, so this change is no longer needed

**Comment by dlom** [comment]
Quick question, why not use the npm version of anesidora instead of directly using the repository as dependency?

**Pull request closed without merging**